packages/web/components/DesktopWindow.tsx

```diff
@@ -16,6 +16,7 @@ interface DesktopWindowProps {
   height?: number | string;
   titleChildren?: React.ReactNode;
   children: React.ReactNode;
+  onResize(): void;
```

Comment:
This give you an error in the useEffect though which is why I switched it. Maybe there is some slick syntax to avoid it? 

![image](https://user-images.githubusercontent.com/46383749/141665870-d6d14dcd-b103-4a62-855e-bbdd59a85c6f.png)

Reply:
you'll have to wrap it in a function like 
useEffect(() => {
  if (onResize) {
    onResize()
  ....